Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.css

```diff
@@ -42,16 +42,25 @@
     color: var(--text-color-secondary);
 }
 
-.details-section > .content > .group > .row.font-variation > .value {
+.details-section > .content > .group > .row.font-variation > input[type="number"] {
     grid-column: axis-value;
-    text-align: end;
+    height: max-content;
+    width: 6ch;
+    padding-block: 4px;
+    padding-inline: 0.5ch;
+    margin-block-start: -6px;
```

Comment:
Maybe we need to adjust the gap instead and add padding to the variation axis tag instead? I'm always mildly suspicious of negative margins and padding 😅

Reply:
I know the feeling :) Adjusting the gap here won't help.

Here's the grid layout for that section without the negative margin in the value input:
<img width="357" alt="Screenshot 2023-03-02 at 5 10 39 PM" src="https://user-images.githubusercontent.com/105809798/222485783-b6c9b5cd-fd0d-42b0-80a9-bfecb90ae5e3.png">

Without the negative margin, the number input is aligned along the block-start with the axis tag and the container for the range input (which is another grid) and its min/max values.

Getting the tag, range input and number input to align to a common baseline is tricky. I might try another grid layout.

The negative margin here is my attempt at ensuring that visual alignment.